frontend/src/components/MapView/LeftPanel/index.tsx

```diff
@@ -72,12 +72,29 @@ const TabPanel = memo(({ children, value, index, ...other }: TabPanelProps) => (
 const LeftPanel = memo(() => {
   const dispatch = useDispatch();
   const tabValue = useSelector(leftPanelTabValueSelector);
+
+  const selectedLayers = useSelector(layersSelector);
```

Comment:
A lot of the changes look like positioning. Can we just make the necessary changes for the fix to make it easier to review? :)

Reply:
the thing is that AAConfig needed AALayerInUrl instead of tabValue ( that was the main change for the fix). So i had to define AALayerInUrl before AAConfig and SelectedLayers before AALayerInUrl.